### PR TITLE
WIP -- How to create a checker

### PR DESCRIPTION
**Pull request by mithro**
Signed-off-by: Tim 'mithro' Ansell <tansell@google.com>

### CODE REVIEW

File: actions/lint/README.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+# `SymbiFlow/actions/checks@main` - GitHub Action for basic SymbiFlow checks.
+
+This GitHub Actions performs a number of basic checks against a repository.
+
+# Using
+
+```yaml
+
+jobs:
+  SymbiFlowBasicChecks:
```
Comment (hzeller) [inline actions/lint/README.md:10]: Looks like a copy-paste relict that wants to be named `VeribleLint` here

File: actions/lint/problem_matcher.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
+{
+    "problemMatcher": [
+        {
+            "owner": "symbiflow-checks",
+            "pattern": [
+                {
+                    "regexp": "^:([^ ]+) file=([^,]+),line=([^,]),col=([^:]+):(.*)$",
+		    "severity": 1,
```
Comment (hzeller) [inline actions/lint/problem_matcher.json:8]: the indentation looks uneven here.

File: actions/lint/problem_matcher.json [hunk 3]
```diff
@@ -0,0 +1,17 @@
+{
+    "problemMatcher": [
+        {
+            "owner": "symbiflow-checks",
```
Comment (hzeller) [inline actions/lint/problem_matcher.json:4]: s/symbiflow/verible/ ?

**Comment by mithro** [comment]
@hzeller -- I don't plan to continue the development of this pull request, it was more just a demo so you could see how it works.

**Pull request closed without merging**